src/listen/websocket.rs

```diff
@@ -366,98 +637,186 @@ where
                 .header("upgrade", "websocket")
                 .header("sec-websocket-version", "13");
 
-            let builder = if let Some(api_key) = self.builder.deepgram.api_key.as_deref() {
-                builder.header("authorization", format!("token {}", api_key))
+            let builder = if let Some(api_key) = builder.deepgram.api_key.as_deref() {
+                http_builder.header("authorization", format!("token {}", api_key))
```

Comment:
Should this be capital T for Token?

Reply:
This way seems to work, but I think it's more usual to capitalize it.  I have updated.